www/docs/en/dev/plugin_ref/spec.md

```diff
@@ -17,700 +17,498 @@ license: >
+
+#### plugins-plist
+
+  Specifies a key and value to append to the correct `AppInfo.plist` file in an iOS Cordova project. This is _outdated_ as it only applies to cordova-ios 2.2.0 and below. Use the `<config-file>` tag for newer versions of Cordova. 
+
+  Example:
+  ```
+  <plugins-plist key="Foo" string="CDVFoo" />
+  ```
+
+#### lib-file
+
+  Like source, resource, and header files, but specifically for platforms such as BlackBerry 10 that use user-generated libraries. For the Windows platform, the `<lib-file>` element allows the inclusion of an `<SDKReference>` in the generated Windows project files.
+
+  Attributes(type) | Description
+  ---------------- | ------------
+  src(string) | *Required* <br/> The location of the file relative to `plugin.xml`. If `src` can't be found, CLI stops and reverses the installation, issues a warning about the problem, and exits with a non-zero code. <br/> For Windows, it indicates the name of the SDK to include (which will be used as value of the `Include` attribute of the generated `<SDKReference>` element).
+  arch(string) | The architecture for which the `.so` file has been built, either `device` or `simulator`. <br/> For Windows, it indicates that the `<SDKReference>` should only be included when building for the specified architecture. Supported values are `x86`, `x64` or `ARM`.
+  device-target(string) | ==windows== <br/> Allowed values: `win` (or `windows`), `phone` or `all`. <br/> Indicates that the `<SDKReference>` should only be included when building for the specified target device type.
+  versions(string) | ==windows== <br/> Indicates that the `<SDKReference>` should only be included when building for versions that match the specified version string. Value can be any valid node semantic version range string.
+
+  Examples:
+  ```
+  <lib-file src="src/BlackBerry10/native/device/libfoo.so" arch="device" />
+  <lib-file src="src/BlackBerry10/native/simulator/libfoo.so" arch="simulator" />
+  ```
+
+  For Windows:
+  ```
+  <lib-file src="Microsoft.WinJS.2.0, Version=1.0" arch="x86" />
+  <lib-file src="Microsoft.WinJS.2.0, Version=1.0" versions=">=8.1" />
+  <lib-file src="Microsoft.WinJS.2.0, Version=1.0" target="phone" />
+  <lib-file src="Microsoft.WinJS.2.0, Version=1.0" target="win" versions="8.0" arch="x86" />
+  ```
+
+#### framework
+
+  Identifies a framework (usually part of the OS/platform) on which the plugin depends.
+
+  Attributes(type) | Description
+  ---------------- | ------------
+  src(string) | *Required* <br/> The name of the system framework or the relative path to one which is included as part of your plugin files.
+  custom(boolean) | Is a boolean indicating whether the framework is one that is included as part of your plugin files.
```

Comment:
For a string of attributes here, the description reads "Is a *type*...". Listing the type again is redundant; they should probably just match the rest of the attributes descriptions.